invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/DocumentsSearch/config.js

```diff
@@ -27,11 +27,14 @@ const sortOrderValues = config.sortOrder.map(sortField => {
 
 const aggsMappings = {
   document_types: 'Document types',
+  items_available_for_loan: 'Available for loan',
```

Comment:
Again, sorry if I am picky, but it is misleading, for me it means:
"Give me all the book that I can loan it at some point"

As a user, I am not sure I understand that having physical copies means I can loan it.
We should be more user friendly:

**Backoffice**
 * Documents with electronic items
 * Documents with physical copies
 * Documents with copies not on loan

**Frontsite** (you forgot to add it!!!):
 * Books with electronic version
 * Books with copies at the library
 * Books that can be loaned now

I am very verbose here, we should find something easier, but do you see my point?

Maybe something with a short title and then an explanation, like for example:

![Screenshot 2019-06-18 at 15 11 55](https://user-images.githubusercontent.com/2089455/59684982-6df2ba00-91db-11e9-9b01-f62fa27f56ba.png)